### ISSUE TITLE
Changed output

### ISSUE THREAD

**Issue by andywer** [SamVerschueren/listr-verbose-renderer#2]
FYI: I just changed the VerboseRenderer's output to better suit npm-launch's needs. Tell me if you want me to open a PR or anything.

**Before**
<img width="521" alt="bildschirmfoto 2016-10-15 um 12 05 19" src="https://cloud.githubusercontent.com/assets/1842462/19409231/fe5afa52-92cf-11e6-8a5d-275eef7628f4.png">

**After**
<img width="517" alt="bildschirmfoto 2016-10-15 um 12 04 21" src="https://cloud.githubusercontent.com/assets/1842462/19409233/06c818e6-92d0-11e6-86ec-ac4df95e10bf.png">

Changes:
- Put the task state in front, so it is aligned
- Add a small figure to each task state
- Show the ancestor tasks if tasks are nested
- Remove the `→` in front of the task's output

Btw: As you can see the stderr output does not get piped through `utils.log()`. Seems like a bug, but not a critical one.

**Comment by SamVerschueren** [comment]
I agree that it looks better. Although, I don't really like the figure inside the task state. I would also go with these states instead.

- `starting`
- `finished`
- `skipped`
- `failed`

If you could come up with `failed` having the same length as the others, feel free :). But this way, the first three are nicely aligned.

**Comment by SamVerschueren** [comment]
@andywer Still interested in doing this?

@okonet I noticed that you have a `verbose` option in `lint-staged`. Do you have any feedback for this? Like to polish the output for this renderer.

**Comment by okonet** [comment]
I'm quite happy with the output for now! I'd use icons instead of words for statues.

**Comment by andywer** [comment]
@SamVerschueren You didn't like the icons in the state brackets. Where would you put them? Or is it the icons themselves?

**Comment by SamVerschueren** [comment]
Maybe we should just show the figures and drop the word entirely.

- starting ▶
- finished ✔
- skipped ↓
- failed ✖

Or would it make sense to keep them? Is that what you meant @okonet ?

**Comment by andywer** [comment]
I thought about that as well. Finished and failed are fine, but starting and skipped... I have my doubts if I was able to tell their meaning just by the icon 😕

**Comment by SamVerschueren** [comment]
Yes, have the same doubts. I looked at [this list](https://github.com/sindresorhus/figures#figures) but couldn't find better icons to be honest. The arrow down is also the one we use in the default renderer to indicate a skipped task.
